workflows/chipseq/chipseq-standard.wdl

```diff
@@ -159,7 +163,16 @@ workflow chipseq_standard {
         use_all_cores=use_all_cores,
         max_retries=max_retries
     }
-    call picard.validate_bam { input: bam=markdup.mkdupbam, max_retries=max_retries }
+    call picard.validate_bam { input: 
+        bam=markdup.mkdupbam,
+        ignore_list=["MISSING_PLATFORM_VALUE",
+                    "INVALID_PLATFORM_VALUE",
+                    "INVALID_MAPPING_QUALITY",
+                    "MATES_ARE_SAME_END",
+                    "MISMATCH_FLAG_MATE_NEG_STRAND",
+                    "MISMATCH_MATE_ALIGNMENT_START"],
```

Comment:
These highlighted 4 feel like things that shouldn't be ignored. Are BAMs really coming out of SEAseq with all these problems? Should that be addressed instead of ignored here?

Reply:
It's an issue with `bowtie`. It looks like `bowtie` is emitting multiple alignments, that have equal alignment scores. Instead of picking one to be primary, it just outputs all of them, without marking any as secondary. Then `picard` fails to match the read records properly. It should be able to use the mate start position information to find the proper mate, but it doesn't. So you get these errors.

Reply:
Interesting. Is this a known and logged bug in `bowtie`? Is the SEAseq team aware this is happening? And is this why you started implementing `samtools fixmate`? Would that fix these errors? (sorry for the bombardment of questions)

I'd much rather fix these errors over simply ignoring them. If I'm understanding correctly, this update (and possible the current version) are outputting BAMs which don't follow the SPEC. I'm not a fan of passing the buck off to `bowtie` and saying "not a problem with our code". This BAMs will likely fail downstream. I don't think I'm comfortable just glossing over these errors.

My two cents: we either need an update/rollback on `bowtie` so this isn't a problem in the first place, or we need a tool like `samtools fixmate` to address the bowtie problems before we can call these BAMs "harmonized".

Reply:
I believe the solution is to use `bowtie2`. I don't think there is any support or development of `bowtie` at this point. I started implementing `samtools fixmate` for a different issue, so I didn't try it here.